approachbouyopen:
  defaults:
    timeout: 20
  alignment_test:
    timeout: 5


centerheaveglyph:
  defaults:
    heave_down_speed: 0.25
    heave_up_speed: 0
    deadband: 1.0
    timeout: 15

centeryawglyph:
  defaults:
    surge_speed: 0.2
    yaw_factor: 0.004
    timeout: 25

approachbuoyclosed:
  defaults:
    timeout: 10
    heave_factor: 0.002 
    yaw_factor: 0.004

fallback:
  defaults:
    timeout: 3

ascend:
  defaults:
    target_depth: 0.25
    timeout: 3

passbouy:
  defaults:
    timeout: 5

findglyph:
  defaults:
    timeout: 7

pause:
  defaults:
    timeout: 3

contingencysubmerge:
  defaults:
    timeout: 15


contingencyapproach:
  defaults:
    timeout: 7


